tests/js/server/utils-tests.js

```diff
@@ -0,0 +1,81 @@
+/*
+
+    Tests for the gpii.express.user.utils component.
+
+ */
+/* eslint-env node */
+"use strict";
+
+var fluid  = require("infusion");
+fluid.logObjectRenderChars = 4096;
+
+var gpii   = fluid.registerNamespace("gpii");
+var jqUnit = require("node-jqunit");
+
+require("../lib/");
+require("../../../src/js/server/lib/datasource");
+
+require("gpii-pouchdb");
+gpii.pouch.loadTestingSupport();
+
+fluid.registerNamespace("gpii.express.user.utils.tests");
+
+fluid.defaults("gpii.express.user.utils.tests.caseHolder", {
+    gradeNames: ["fluid.test.testCaseHolder"],
```

Comment:
You aren't using a caseHolder grade that knows to wait for pouch to startup (such as one from gpii-pouchb), so it might be trying to access pouch before it's had a chance to start.  You can either use prior art in the gpii-pouchdb test fixtures, or wire up sequence steps that will cause the first actual test to only be run once pouch has started up.